src/main/resources/view/ExercisesPanel.fxml

```diff
@@ -3,6 +3,7 @@
 <?import javafx.scene.control.TextArea?>
 <?import javafx.scene.layout.StackPane?>
 
+<!-- //@@author iNekox3 -->
```

Comment:
Do we need to add this line for our code also?

Reply:
Only required starting from v1.4, but definitely will be easier to keep track of what you've already done if you start now!